contracts/governance/Staking/modules/shared/StakingShared.sol

```diff
@@ -70,6 +70,7 @@ contract StakingShared is StakingStorageShared, SafeMath96 {
         // abstract
     }
 
+    //SHOULDN'T BE USED, CAN BE EASILY BROKEN USING CONTRACT
```

Comment:
why not just remove the function to avoid misuse?

Reply:
The answer is quite simple. 
Don't you consider this function can be used in another places?
It's out of scope for this task, should be done as a separate task.

Reply:
The answer is quite simple.
Don't you think this function can be used in other places? :-)
It's out of scope for this task, and should be done as a separate task.

Reply:
ok. then pls restate the comment to add more clarity. otherwise it shouldn't be used but in fact it is.

Reply:
We will have a separate story for it.